### DISCUSSION TITLE
Is there a reason that the HotChocolate.Stitching package no longer receives preview updates?

### DISCUSSION THREAD

**Discussion by MortenChristiansen** [ChilliCream/hotchocolate#5555] [General]
I have recently started experimenting with GraphQL and Hot Chocolate. I got the impression that v13 is not too far off, so I decided to use this version so as to not have to figure out a new version later which has served me well enough so far. Today I took a look at the schema federation stuff but ran into the problem that the HotChocolate.Stitching package is stuck at preview 24. This makes it incompatible with all the other packages, preventing me from referencing it and continuing on the schema federation. I wonder if it is an oversight that this package has not been updated or if there is some technical issue preventing it?

**Comment by tobias-tengler** [comment]
Initially it was planned to retire the old stitching library in favor of a improved solution, dubbed Fusion, in v13. It won't be done in time for the v13 release though, so the current plan is to "re-add" the old stichting library and make it compatible with v13. If you look at the PRs there should be one that is working on that. I think you can expect the package to release previews again in the near future and be part of v13. Fusion should arrive in a dot release next year.
Here's the announcement, if you're part of our community Slack channel: https://hotchocolategraphql.slack.com/archives/C01EZSSF6NS/p1666892750899799?thread_ts=1666892750.899799&cid=C01EZSSF6NS

**Reply by MortenChristiansen** [reply]
Cool, I'll be looking forward to that :) Thanks
